drghs-worker/maintnerd/main.go

```diff
@@ -159,7 +194,13 @@ func main() {
 
 	group.Go(func() error {
 		// Add gRPC service for v1beta1
-		grpcServer := grpc.NewServer(grpc.UnaryInterceptor(unaryInterceptorLog))
+		grpcServer := grpc.NewServer(
+			grpc.UnaryInterceptor(
+				grpc_middleware.ChainUnaryServer(
+					grpctrace.UnaryServerInterceptor(global.Tracer("maintnerd")),
```

Comment:
Is there a way to improve how the spans are labeled. Right now in the Trace UI, the top-level GetIssue span looks like it's calling another GetIssue span, they're not labeled as "maintner-rtr" and "maintnerd"

Reply:
I think this might be possible by adding a custom DisplayNameFormatter in the Stackdriver Exporter 
https://godoc.org/github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/trace#WithDisplayNameFormatter

Reply:
Good enough for now I guess